### PR TITLE
8763 prevent parties from being served case documents before the petition is served

### PR DESCRIPTION
**Pull request by kkoskelin**
Also split some tests for document viewer helper into second file.

Screen Shot 2021-08-24 at 4 21 24 PM
Screen Shot 2021-08-25 at 10 54 46 AM

### CODE REVIEW

**Comment by matthopson** [comment]
Are we addressing any of the backend on this? I worked on a similar bug related to this (that was ultimately closed), but did more to try and address it from the backend.

https://github.com/ustaxcourt/ef-cms/pull/1213/files

File: web-client/src/presenter/computeds/addCourtIssuedDocketEntryHelper.test.js [hunk 1]
```diff
@@ -224,12 +225,40 @@ describe('addCourtIssuedDocketEntryHelper', () => {
...
+          eventCode: 'O',
+        },
+      },
+    });
+    expect(result.showServiceWarning).toEqual(true);
+    expect(result.showSaveAndServeButton).toEqual(false);
+  });
+
+  it('should return showServiceWarning false and showSaveAndServeButton true if eventCode is NOT found in unservable event codes list and petition on case has been served', () => {
+    const result = runCompute(addCourtIssuedDocketEntryHelper, {
```
Comment (rachelschneiderman) [inline web-client/src/presenter/computeds/addCourtIssuedDocketEntryHelper.test.js:252]: Nit: I don't see an expect in this test for 'showServiceWarning' being false

Comment (kkoskelin) [reply web-client/src/presenter/computeds/addCourtIssuedDocketEntryHelper.test.js:252]: Darn it, good catch. Addressing. thank you.

**Comment by kkoskelin** [comment]
@matthopson Thanks for your observation and link to your closed PR.  Added a task to this Story.  (this PR is one of several for this story)

**Review by ericsorenson — APPROVED** [review]

**Pull request merged**